unearthed-arcana/2021/20211008.xml

```diff
@@ -0,0 +1,551 @@
+		<supports>UA20211008 Astral Fire</supports>
+		<description>
+			<p>Your spellcasting ability for chosen cantrip is Wisdom.</p>
+		</description>
+		<sheet display="false">
+			<description>Your spellcasting ability for chosen cantrip is Wisdom.</description>
+		</sheet>
+		<rules>
+			<stat name="ua:astral fire:spellcasting" value="Wisdom" inline="true" />
+		</rules>
+	</element>
+	<element name="Charisma" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_ASTRAL_FIRE_CHARISMA">
+		<supports>UA20211008 Astral Fire</supports>
+		<description>
+			<p>Your spellcasting ability for chosen cantrip is Charisma.</p>
+		</description>
+		<sheet display="false">
+			<description>Your spellcasting ability for chosen cantrip is Charisma.</description>
+		</sheet>
+		<rules>
+			<stat name="ua:astral fire:spellcasting" value="Charisma" inline="true" />
+		</rules>
+	</element>
+
+	<element name="Fey Ancestry" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_FEY_ANCESTRY">
+		<description>
+			<p>You have advantage on saving throws you make to avoid or end the charmed condition on yourself.</p>
+		</description>
+		<sheet>
+			<description>You have advantage on saving throws you make to avoid or end the charmed condition on yourself.</description>
+		</sheet>
+	</element>
+
+	<element name="Keen Senses" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_KEEN_SENSES">
+		<description>
+			<p>You have proficiency in the Perception skill.</p>
+		</description>
+		<sheet display="false">
+			<description>You have proficiency in the Perception skill.</description>
+		</sheet>
+		<rules>
+			<grant type="Proficiency" id="ID_PROFICIENCY_SKILL_PERCEPTION" />
+		</rules>
+	</element>
+
+	<element name="Radiant Soul" type="Racial Trait" source="Unearthed Arcana: Travelers of the Multiverse" id="ID_WOTC_UA20211008_RACIAL_TRAIT_ASTRAL_ELF_RADIANT_SOUL">
+		<description>
+			<p>When you succeed on a death save, you can regain a number of hit points equal to your proficiency bonus + your Intelligence, Wisdom, or Charisma modifier (choose when you select this race). You can’t use this trait again until you finish a long rest.</p>
+		</description>
+		<sheet usage="1/Long Rest">
+			<description>When you succeed on a death save, you can regain {{ua:radiant soul}} hit points.</description>
+		</sheet>
+		<rules>
+			<!-- <select type="Racial Trait" name="Ability Score (Radiant Soul)" supports="UA20211008 Radiant Soul" /> -->
+			<stat name="ua:radiant soul" value="proficiency" bonus="base" />
+			<stat name="ua:radiant soul" value="intelligence:modifier" bonus="ability" />
+			<stat name="ua:radiant soul" value="wisdom:modifier" bonus="ability" />
+			<stat name="ua:radiant soul" value="charisma:modifier" bonus="ability" />
```

Comment:
Based on the description, I believe that it should use the same the same stat as Astral Fire